.github/workflows/update-docs.yml

```diff
@@ -0,0 +1,38 @@
+---
+name: Update-docs
+on:              # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - main
+jobs:
+  transfer-file:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Pushes Readme file
+        uses: dmnemec/copy_file_to_another_repo_action@main
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          API_TOKEN_GITHUB: ${{ secrets.ARC_DOCS_API_TOKEN_GITHUB }}
+        with:
+          source_file: 'README.md'
+          destination_repo: 'sourcefuse/arc-docs'
+          destination_folder: 'docs/arc-iac-docs/modules/arc-backstage-ecs-app'
```

Comment:
if i am not wrong then the destination folder should be like this
`docs/arc-iac-docs/modules/arc-lambda`